tests/integration/steps/integration_steps.go

```diff
@@ -0,0 +1,622 @@
+package steps
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	resty "github.com/go-resty/resty/v2"
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const RETRY_TIMEOUT = 15 * time.Minute
+const RETRY_INTERVAL = 30 * time.Second
+
+var namespace string
+var repairId string
+var releaseName string
+
+var (
+	Info    = Color("\033[1;33m%s\033[0m")
+	Outline = Color("\033[1;34m%s\033[0m")
+)
+
+func Color(colorString string) func(...interface{}) string {
+	sprint := func(args ...interface{}) string {
+		return fmt.Sprintf(colorString,
+			fmt.Sprint(args...))
+	}
+	return sprint
+}
+
+func timeTrack(start time.Time, name string) {
+	elapsed := time.Since(start)
+	log.Println(Info(fmt.Sprintf("%s took %s", name, elapsed)))
+}
+
+func RunShellCommand(command *exec.Cmd) error {
+	err := command.Run()
+	return err
+}
+
+func RunShellCommandAndGetOutput(command *exec.Cmd) string {
+	var outb bytes.Buffer
+	command.Stdout = &outb
+	err := command.Run()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	return string(outb.String())
+}
+
+// Find returns true if val exists in the slice array, false otherwise
+func Find(slice []string, val string) bool {
+	for _, item := range slice {
+		if item == val {
+			return true
+		}
+	}
+	return false
+}
+
+func g(t *testing.T) *gomega.WithT {
+	return gomega.NewWithT(t)
+}
+
+func deployCluster(t *testing.T, customValues string, helmValues map[string]string) {
+	clusterChartPath, err := filepath.Abs("../../charts/k8ssandra")
+	g(t).Expect(err).To(gomega.BeNil())
```

Comment:
I guess you want to have ``.`` in the import segment for the gomega. This way the ``gomega.BeNil()`` can be written as ``BeNil()``, which makes sense for a matcher library (kinda like Java's static import)

Reply:
Right, I had an issue initially with the dot import when invoking `NewWithT()` but I must have messed up as it's working now.
Good call 👍

Reply:
Done